src/components/MetadataForm.jsx

```diff
@@ -102,7 +98,7 @@ class MetadataForm extends Component {
             label="Select a role"
             name="role"
             value={this.state.role}
-            onChange={(e) => this.handleInputChange(e)}
+            onChange={this.handleInputChange}
```

Comment:
tiny change here, this seems to break the `SelectInput` fields (but the change works for the other inputs)